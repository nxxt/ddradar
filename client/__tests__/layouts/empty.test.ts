import { createLocalVue, shallowMount } from '@vue/test-utils'

import EmptyLayout from '~/layouts/empty.vue'

const localVue = createLocalVue()

describe('layouts/empty.vue', () => {
  test('renders correctly', () => {
    const wrapper = shallowMount(EmptyLayout, {
      localVue,
      stubs: { Nuxt: true },
    })
    expect(wrapper).toMatchSnapshot()
  })
})
