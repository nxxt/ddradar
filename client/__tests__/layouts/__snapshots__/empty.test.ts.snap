// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layouts/empty.vue renders correctly 1`] = `<nuxt-stub></nuxt-stub>`;
